### PR TITLE
Make Response page a view and reallocate data appropriately

### PR DESCRIPTION
**Pull request by laytonmaes**
<h1>PULL REQUEST</h1>

<ul>
  <li>What does it change?</li>
Make Response page its own view and move data to that view.
  <li>What does it fix?</li>
makes things components good
</ul>

<h2>Type of Change</h2>
refactor
_Delete options that are not relevant_

- [x] **New Change**

<h2>How has this been tested?</h2>

- [x] **JS**
- [x] **Dev Tools**
- [x] **Website**

<h2>Checklist</h2>

- [x] My code follows the style guidelines of this project
- [x] I have performed a self-review of my own code
- [x] I have commented my code, particularly in hard-to-understand areas
- [x] My changes generate no new bugs
- [x] I have tagged project partners for review & it has been reviewed 

<h2>Does this PR close any issues?</h2>

### CODE REVIEW

File: App.js [hunk 1]
```diff
@@ -8,63 +8,32 @@ import Results from './components/Results';
...
-  const [suggestions, setSuggestions] = useState(Data().suggestions)
-  const [resultsCards, setResultsCards] = useState("")
-  const allResultsCards = makeArray(suggestions)
+
 
   const Stack = createNativeStackNavigator();
 
   const getView = (uri) => {
-    createImageComponents([...allImages, <PlantImage uri={uri}/>])
+    createImageComponents([...allImages, <PlantImage key={uri} uri={uri}/>])
```
Comment (BlueNealis) [inline App.js:30]: Yay you added a key :D

File: Views/data2.js [hunk 2]
```diff
@@ -0,0 +1,195 @@
+import * as React from 'react';
+import { Text, View, StyleSheet, Image } from 'react-native';
+
+export default function Data2() {
```
Comment (BlueNealis) [inline Views/data2.js:4]: I do like these place-holder pieces of data but I wonder if we should migrate before deployment to a loading animation or skeleton screen?

**Review by BlueNealis — APPROVED** [review]
I like these changes, easy to read through

**Review by Neil-B-Linton — APPROVED** [review]
Some awesome beautiful code! I love it!

**Review by ameliaeiger — APPROVED** [review]

**Pull request merged**